### ISSUE TITLE
Code of now playing model

### ISSUE THREAD

**Issue by krishvishal** [mimbres/neural-audio-fp#11]
Is it possible include code for the Google's now playing model here? Even if it's not functional.

**Comment by mimbres** [comment]
Yes, it's planned. 
But I didn't include a config file for that, because I need to test if the triplet loss works correctly after migrating to this open project. Actually the [triplet loss function](https://github.com/mimbres/neural-audio fp/blob/main/model/fp/online_triplet_loss.py) is exactly for that.
 
- [ ] unit-test triplet loss 
- [ ] fix train.py, ,generate.py
- [ ] write a config template for the now-playing mode: 2 sec input (original) and 1sec input for modified version.
- [x] unit-test nnfp.py (done)
- [ ] test-drive of training

Not today, but I will make it soon!

**Comment by guillemcortes** [comment]
Hi @mimbres sorry to bring this up, what's the actual status of the project? Are you still working on it? Do you plan to release this code?

**Comment by mimbres** [comment]
@guillemcortes  Sorry, somehow I forgot about this. I'll update it in a week.

**Comment by krishvishal** [comment]
@mimbres is the code uploaded?

**Comment by mimbres** [comment]
@krishvishal No, it's still being delayed. I'll notice when updated.

**Comment by mimbres** [comment]
@krishvishal @guillemcortes `Now-playing` model with `Online (semi-hard) triplet loss` is now available.
```python3
python run.py train CHECKPOINT_NAME -c now_playing
```
Results and pre-trained model will be available in this week.
